vm-setup/roles/packages_installation/tasks/ubuntu_required_packages.yml

```diff
@@ -0,0 +1,105 @@
+---
+- name: Install required Ubuntu packages
+  block:
+    - name: Update all packages to their latest version
+      apt: update_cache=yes name="*" state=latest
+
+    - name: Fetch yarn gpg key
+      apt_key:
+        url: https://dl.yarnpkg.com/debian/pubkey.gpg
+        state: present
+
+    - name: Add yarn release key
+      lineinfile:
+        path: /etc/apt/sources.list.d/yarn.list
+        line: deb https://dl.yarnpkg.com/debian/ stable main
+        create: yes
+        owner: root
+        group: root
+        mode: '0644'
+
+    - name: Add OS release key
+      lineinfile:
+        path: /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+        line: "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_{{ OS_VERSION_ID }}/ /"
```

Comment:
would be good for visibility if we could break the long line

Reply:
I did in my local tests, and unfortunately was failing. I would rather keep it as it is now. And it is a URL link, that's why difficult to break.